Dockerfile

```diff
@@ -8,7 +8,7 @@ WORKDIR /app
 COPY . /app
 
 # Install package.json dependencies (yes, clean up must be part of same RUN command because of layering)
-RUN apk add --no-cache gcompat
+RUN apk add --no-cache gcompat curl
 RUN apk add --update python build-base git && npm install && apk del python build-base && rm -rf /var/cache/apk/*
```

Comment:
Currently compact and curl versions can be out of date since --update option is not used.
With recent version of Alpine you can do:
`RUN apk add --update --no-cache [list-of-packages] && ...`
--no-cache option does `rm -rf /var/cache/apk`

Also I don't think that Git is needed after build so a multi stage build would be better here.